sleap/gui/commands.py

```diff
@@ -2725,6 +2728,22 @@ def do_action(context, params):
         # Now add the selected instance to the `InstanceGroup`
         context.execute(SetSelectedInstanceGroup, instance_group=instance_group)
 
+#TODO: Add class for changing the name of an instance group
+class SetInstanceGroupName(EditCommand):
+
+    @staticmethod
+    def do_action(context: CommandContext, params: dict):
+        instance_group = params["instance_group"]
+        name = params["name"]
+
+        # Access the name registry through the frame group of the instance group
+        frame_group = context.state["session"].frame_groups[instance_group.frame_idx]
+        name_registry = frame_group._instance_group_name_registry
+
+        if name in name_registry:
+            raise ValueError(f"Name {name} already in use. Please use a unique name.")
```

Comment:
We don't need this since it is already handled-in/copied-from `InstanceGroup.set_name`
```suggestion
```
Having this here would just call double the same code if the `name` is not in the `name_registry` or call the same code if the `name` is in the `name_registry`.

Reply:
Weird class structure... use `FrameGroup.set_instance_group_name`:
https://github.com/talmolab/sleap/blob/a32d68a1c116584c8950fde841310e77cc71ac99/sleap/io/cameras.py#L1882-L1889